### DISCUSSION TITLE
github page generates buttons in my html that I never added.

### DISCUSSION THREAD

**Discussion by Christian-Cachero** [community/community#62254] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug GitHub Pages

### Body

So I have this simple static web page, everything works as it should but I have a problem that I can not find a solution I have to adminit that had happened to me before a few months ago but I do not remember how to solve it, the thing is that today I applied some manual changes to my code to add responsive design, and do not change much more than some things of Boostrap and update the stylesheet, and now again I have this problem and I'm going crazy please help me.

displayed page where this is happening:
https://christian-cachero.github.io/CaC-Integrador-Boostrap/

source code of the same page:
https://github.com/christian-cachero/CaC-Integrador-Boostrap/blob/master/index.html

screenshots:
https://i.imgur.com/RX05CYS.png
https://i.imgur.com/fVfEH63.png

if anyone knows the cause of this problem I would be very grateful if you could tell me what I need to do.

**Comment by bronglil** [comment]
If GitHub Pages is generating buttons in your HTML that you never added, it's possible that the issue might be related to the theme you're using or some default settings. Here are steps to help you troubleshoot and resolve the problem:

Check Theme Settings:

**Reply by Christian-Cachero** [reply]
Excuse me, which theme options?
